app/(afterLogin)/(planningAndReporting)/planning-and-reporting/_components/reporting/index.tsx

```
@@ -256,10 +256,35 @@
         {allReporting?.items?.map((dataItem: any, index: number) => (
           <>
             <Card
-              className="mb-2"
+              bodyStyle={{ padding: '12px' }}
+              headStyle={{ borderBottom: 'none' }}
+              className="mb-1 bg-[#fafafa]"
               key={index}
               title={
                 <div>
+                   <Row className="flex justify-start mb-1 ">
```

ESLint

[error] Delete `·`

(prettier/prettier)